gcloud/storage/__init__.py

```diff
@@ -41,11 +41,11 @@
 import os
 
 from gcloud import credentials
+from gcloud._helpers import get_default_project
```

Comment:
Note, `gcloud._helpers` comes before `gcloud.connection` alphabetically here, but we don't hold to any rules, so no need to change anything.